docs/UPGRADE-2.0.md

```
@@ -11,6 +11,9 @@ This guide provides detailed instructions for upgrading from `linkxtr/laravel-qr
     - [1. PHP Version Requirement](#1-php-version-requirement)
     - [2. Namespace Changes](#2-namespace-changes)
     - [3. Color System](#3-color-system)
+    - [4. Strict Data Validation](#4-strict-data-validation)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)